srcpkgs/chroot-curl/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'chroot-curl'
+pkgname=chroot-curl
+version=7.79.1
+revision=1
+bootstrap=yes
```

Comment:
This should not be flagged as bootstrap=yes.

Reply:
Then this package could be a sub-package of `curl`, no?

Reply:
Would require two builds done in the same template, which is kind of ugly...

Reply:
Here we are also omitting the more complicated dependencies.